client/java/src/main/java/com/linecorp/centraldogma/client/AbstractWatcher.java

```diff
@@ -317,6 +362,17 @@ private void notifyListeners(Latest<T> latest) {
         }
     }
 
+    private void updateLatestReceivedTime() {
+        if (meterRegistry == null) {
+            return;
+        }
+
+        if (latestReceivedTime.getAndSet(Instant.now().getEpochSecond()) == 0) {
+            // emit metrics once the values are ready
+            meterRegistry.gauge(LATEST_RECEIVED_TIME_METER_NAME, tags, latestReceivedTime, AtomicLong::get);
```

Comment:
Question) Will this actually register `latestReceivedTime` subsequently after the first registration since the `Meter.Id` is already registered? Would it make more sense to register the gauge once and define the update function to fetch `AbstractWatcher#latestReceivedTime` instead?